### PR TITLE
add color token of audio state in participant List

### PR DESCRIPTION
**Pull request by HuanhuanHou**
# Guide

**This "Help" section can be deleted before submitting this pull request.**

*Update the name of this pull request to reflect the following shape:*

```
{type}/{scope?}/{message}
```

* **type** - A [conventional commit type](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum) **REQUIRED**
* **scope** - The kabab-case scope of the changes in this request
* **message** - add color token of audio state in participant List **REQUIRED**

*Provide a general summary of the scope of the changes in this pull request.*

* [Readme](https://github.com/momentum-design/tokens/blob/master/README.md)
* [Getting Started Guide](https://github.com/momentum-design/tokens/blob/master/GETTING_STARTED.md)
* [Contributing Guide](https://github.com/momentum-design/tokens/blob/master/CONTRIBUTING.md)

# Description

*The full description of the changes made in this request.*

# Links

*Links to relevent resources.*

### CODE REVIEW

File: platformcomponents/desktop/participant-list.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        "background": "@theme-indicator-attention"
+      }
+    },
+    "audioUnmuted": {
+      "#normal": {
+        "background": "@theme-indicator-stable"
+      }
+    }
+  }
+}
```
Comment (HuanhuanHou) [inline platformcomponents/desktop/participant-list.json:16]: @NotNestor , It seems I can't add reviewer. Could you pls help to have a review?

File: platformcomponents/desktop/participant-list.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "participantList": {
+    "comment": "controls at participantList panel, mute/unmute, etc.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=9332%3A2007",
+    "audioMuted": {
+      "#normal": {
+        "background": "@theme-indicator-attention"
```
Comment (nataliadelmar) [inline platformcomponents/desktop/participant-list.json:7]: We do not need the 'background' layer. You can keep it in:
```
"audioMuted":{
     #normal: "@theme-indicator-attention"
}
```
Same for audioUnmuted

File: platformcomponents/desktop/participant-list.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "participantList": {
+    "comment": "controls at participantList panel, mute/unmute, etc.",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/participant-list.json:3]: Capital 'C' in 'Controls' please :)

**Review by nataliadelmar — CHANGES_REQUESTED** [review]
LGTM, asking for small minor changes before merging

File: platformcomponents/desktop/participant-list.json (hunk 2 above)
Comment (HuanhuanHou) [reply platformcomponents/desktop/participant-list.json:7]: @nataliadelmar ，I am not sure if the sub-layer is in need when only a color-theme will be applied this component. I am interested when we should add the sub-layer (such as background and hover, etc)

File: platformcomponents/desktop/participant-list.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "participantList": {
```
Comment (nataliadelmar) [inline platformcomponents/desktop/participant-list.json:2]: 'participantListItem' is the right naming for this component. The name of the file should also change to 'participant-list-item'

**Review by nataliadelmar — CHANGES_REQUESTED** [review]
One more thing and we are ready to go

File: platformcomponents/desktop/participant-list.json (hunk 4 above)
Comment (HuanhuanHou) [reply platformcomponents/desktop/participant-list.json:2]: participantListItem sounds be better.

**Review by nataliadelmar — APPROVED** [review]

**Pull request merged**